modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelper.java

```diff
@@ -730,6 +687,59 @@ protected void validateFieldWithDDMFormFieldValidation(
 		defaultDDMFormFieldValueAccessor =
 			new DefaultDDMFormFieldValueAccessor();
 
+	private void _evaluateDDMFormRule(DDMFormRule ddmFormRule) {
+		if (evaluateDDMFormRuleCondition(ddmFormRule.getCondition())) {
```

Comment:
@rijema, this check is still necessary once we're performing it on lines 733 and 734? If not, can you please find out a way to prevent this repetition?

Reply:
Done.